.github/workflows/flaky-tests.yaml

```diff
@@ -6,6 +6,12 @@ on:
     paths:
       - '**.go'
       - '.github/workflows/flaky-tests.yaml'
+  issue_comment:
+    types: [ created, edited, deleted ]
```

Comment:
Not sure how I feel about this yet. This will probably cause unnecessary runs. Maybe we just re-run the job manually if we need to add an exclude comment.

Reply:
Yeah I agree. If we this on every comment update, it will increase our spend even further. I think rerunning the job manually if ok, it's going to be an exceptional case anyway and shouldn't happen often.